### PR TITLE
Added Apartment Struct + mapping of the apartments and implemented ge…

### PR DESCRIPTION
**Pull request by bpolitiadis**
Added the Apartment metadata struct + mapping of the apartments and implemented get ApartmentDetails function
# Pull Request

## Submission Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor/Cosmetic (non-breaking change which adds readability and/or simplifies the codebase)

## Checklist

- [ ] My PR has an accurate/descriptive/correct name
- [ ] My code follows the [Solidity Style Guide](https://docs.soliditylang.org/en/v0.8.7/style-guide.html)
- [ ] My code has accompanying tests to cover my changes (if needed, refactor is exempt and bugfixes **may** also be)
- [ ] All new and existing tests passed
- [ ] When merging this PR the commit message follows conventional-commits, e.g 'feat: add some feature'

### CODE REVIEW

File: contracts/EncodeDAOCore.sol [hunk 1]
```diff
@@ -25,6 +25,7 @@ contract EncodeDAOCore is ERC721URIStorage, AccessControl {
     Issue[] private acceptedIssues;
     Issue[] private rejectedIssues;
     Counters.Counter private _issueIds;
+    mapping(uint256 => Apartment) apartments;
```
Comment (redrocket94) [inline contracts/EncodeDAOCore.sol:28]:
```suggestion
    mapping(uint256 => Apartment) idToApartment;
```

File: contracts/EncodeDAOCore.sol [hunk 2]
```diff
@@ -41,6 +42,13 @@ contract EncodeDAOCore is ERC721URIStorage, AccessControl {
         IssueStatus status;
     }
 
+    struct Apartment {
+        uint id;
+        uint floor;
+        uint squareMeters;
+        bool heating;
+    }
+
```
Comment (redrocket94) [inline contracts/EncodeDAOCore.sol:51]: 👍

File: contracts/EncodeDAOCore.sol [hunk 3]
```diff
@@ -83,8 +91,9 @@ contract EncodeDAOCore is ERC721URIStorage, AccessControl {
     function getApartmentList() public view {}
 
     /// Get the details of an apartment you own.
-    function getApartmentDetails(uint256 apartmentId) public view {}
-
+    function getApartmentDetails(uint256 apartmentId) public view returns (Apartment apartment) {
+        return (apartments[apartmentId]);
+    }
```
Comment (redrocket94) [inline contracts/EncodeDAOCore.sol:96]:
```suggestion
    function getApartmentDetails(uint256 apartmentId) public view returns (Apartment apartment) {
        return (idToApartment[apartmentId]);
    }
```

**Review by redrocket94 — APPROVED** [review]
Approved with some suggested changes for naming

File: contracts/EncodeDAOCore.sol (hunk 2 above)
Comment (twlyp) [reply contracts/EncodeDAOCore.sol:51]: :+1:

**Review by twlyp — APPROVED** [review]

**Pull request merged**